.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "geotrellis-site"]
```

Comment:
Potentially, although that would break current `docker-compose build` behaviour.